File: spec/code_samples/Javascript_jquery/list@cleaned@{list_name}/delete.js
var settings = {
  "async": true,
  "crossDomain": true,
  "url": "https://analyzemail.com/api/v1/list/cleaned/%7Blist_name%7D?username=SOME_STRING_VALUE",
  "method": "DELETE",
  "headers": {
    "content-type": "application/json"
  }
}

$.ajax(settings).done(function (response) {
  console.log(response);
});

File: spec/code_samples/Python_requests/list@cleaned@{list_name}/delete.py
import requests

url = "https://analyzemail.com/api/v1/list/cleaned/%7Blist_name%7D"

querystring = {"username":"SOME_STRING_VALUE"}

headers = {'content-type': 'application/json'}

response = requests.request("DELETE", url, headers=headers, params=querystring)

print(response.text)